### DISCUSSION TITLE
How to add assersion message

### DISCUSSION THREAD

**Discussion by iyangdidi** [cypress-io/cypress#14486] [Questions and Help]
I need to add customized assersion message to make the failure more readable.
According to the documents https://docs.cypress.io/api/commands/should.html#Function
`expect($lis.eq(0), 'first item').to.contain('Walk the dog')` will works
But the message will only shown when using 'cypress open' not 'cypress run'

'cypress open' 
![image](https://user-images.githubusercontent.com/16665034/104210047-68ce0f00-5400-11eb-9b15-f734a9dd7b36.png)

'cypress run'
![image](https://user-images.githubusercontent.com/16665034/104210098-7a171b80-5400-11eb-84f3-9ddd06ca7ad1.png)

**Comment by estefafdez** [comment]
I think the problem is the element you are getting and not the assertion. 

Is there any other way to get the element and be more precise? If you can share the source-code at least, maybe I could try to help you.

**Reply by iyangdidi** [reply]
@estefafdez 
Thanks, seems like it's an issue and Cypress team is working on it https://github.com/cypress-io/cypress/issues/14484
